api-reference/10 UI Components/dxOverlay/1 Configuration/wrapperAttr.md

```diff
@@ -5,7 +5,110 @@ default: {}
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes" target="_blank">global attributes</a> for the UI component's wrapper element. The wrapper is the first element in the UI component's container and ususally has the `dx-overlay-wrapper` class.
```

Comment:
> The wrapper is the first element in the UI component's container and ususally has the `dx-overlay-wrapper` class.

Let's move this to the full description.

Can we show the resulting markup and indicate where the atributes will be put, where the content will be, etc.?

Reply:
@RomanTsukanov How about giving a screenshot and/or a link to the codesandbox?